src/status_im/contexts/chat/messenger/composer/view.cljs

```diff
@@ -172,12 +172,10 @@
 
 (defn composer
   [props]
-  (let [{:keys [chat-id
-                contact-request-state
-                group-chat
-                able-to-send-message?]
-         :as   chat} (rf/sub [:chats/current-chat-chat-view])]
-    (when (seq chat)
+  (let [current-chat-id       (rf/sub [:chats/current-chat-id])
+        able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
+    (when (seq current-chat-id)
```

Comment:
current-chat-id is a string
We can use `(when-not (string/blank? current-chat-id)`

Reply:
You can also use `seq` to check if a string is empty as well. It's more compact and don't have to require `string`

Reply:
But it's not ideal, As `string/blank?` is faster.
CC: @flexsurfer

Reply:
I think the convention here is to stick with it for strings.

Reply:
wasn't aware of it being faster. Since this is happening in the chat it's better to go with the quicker one. Thanks @ibrkhalil